src/components/modals/StakingModal.tsx

```diff
@@ -307,22 +277,22 @@ const StakingModal = ({
             {tab == 0 &&
               gateway &&
               (amountToStake?.length > 0 ||
-                balances.io < minRequiredStakeToAdd ||
+                (balances?.io || 0) < minRequiredStakeToAdd ||
                 !allowDelegatedStaking) &&
               (errorMessages.cannotStake || errorMessages.stakeAmount) && (
                 <ErrorMessageIcon
                   errorMessage={
                     errorMessages.cannotStake ?? errorMessages.stakeAmount!
                   }
-                  tooltipPadding={"3"}
+                  tooltipPadding={'3'}
                 />
               )}
             {tab == 1 &&
               amountToUnstake?.length > 0 &&
               errorMessages.unstakeAmount && (
                 <ErrorMessageIcon
                   errorMessage={errorMessages.unstakeAmount}
-                  tooltipPadding={"3"}
+                  tooltipPadding={'3'}
```

Comment:
note: not specifying px can cause issues on firefox

Reply:
I see no difference when testing in firefox here